<launch>

    <arg name="marker1Id"         default="0"/>
    <arg name="marker2Id"         default="1"/>
    <arg name="markerSize"        default="0.2"/>    <!-- in m -->
    <arg name="eye"               default="left"/>
    <arg name="dct_normalization" default="False" />
    <arg name="dct_filter_size"   default="2" />
    <arg name="marker1_frame"     default="frame0" />
    <arg name="marker2_frame"     default="frame1" />


    <node pkg="aruco_ros" type="double" name="aruco_simple">    
        <remap from="/camera_info" to="/camera_link/color/camera_info" />
        <remap from="/image" to="/camera_link/color/image_raw" />
        <param name="image_is_rectified" value="True"/>
        <param name="marker_size" value="$(arg markerSize)"/>
        <param name="marker_id1" value="$(arg marker1Id)"/>
        <param name="marker_id2" value="$(arg marker2Id)"/>
        <param name="normalizeImage" value="$(arg dct_normalization)"/>
        <param name="dct_components_to_remove" value="$(arg dct_filter_size)"/>
        <param name="parent_name" value="camera_link"/>
        <param name="child_name1" value="$(arg marker1_frame)" />
        <param name="child_name2" value="$(arg marker2_frame)" />
    </node>

</launch>

